lib/scala/project-manager/src/main/scala/org/enso/projectmanager/infrastructure/languageserver/LanguageServerController.scala

```diff
@@ -79,15 +79,16 @@ class LanguageServerController(
 
   private val descriptor =
     LanguageServerDescriptor(
-      name                           = s"language-server-${project.id}",
-      rootId                         = UUID.randomUUID(),
-      rootPath                       = project.path.toString,
-      projectId                      = project.id,
-      networkConfig                  = networkConfig,
-      distributionConfiguration      = distributionConfiguration,
-      engineVersion                  = engineVersion,
-      jvmSettings                    = distributionConfiguration.defaultJVMSettings,
-      jvmMode                        = processConfig.jvmMode,
+      name                      = s"language-server-${project.id}",
+      rootId                    = UUID.randomUUID(),
+      rootPath                  = project.path.toString,
+      projectId                 = project.id,
+      networkConfig             = networkConfig,
+      distributionConfiguration = distributionConfiguration,
+      engineVersion             = engineVersion,
+      jvmSettings               = distributionConfiguration.defaultJVMSettings,
+      jvmModeEnabled =
+        processConfig.jvmMode || project.jvmModeEnabled.getOrElse(false),
```

Comment:
This is where the tri-state gets converted to binary state, right? Is `LanguageServerController` the right place to encode the default value? Shouldn't there be `project.isJvmMode(): Boolean`?